xtask/src/examples.rs

```diff
@@ -239,6 +240,25 @@ pub fn build_oak_functions_server_variants(opt: &BuildServerOpt) -> Step {
     }
 }
 
+/// Build every variant of the function server.
```

Comment:
This comment does not seem to match the single-step nature of the implementation.

Reply:
Yeah I'm planning to clean this up in an upcoming PR (removing the enum entirely).